GETTING_STARTED.md

```diff
@@ -46,7 +52,7 @@ RSpec.configure do |config|
 end
 ```
 
-Remember to require the above file in your rails_helper since the support folder isn't eagerly loaded
```

Comment:
What do you think about getting rid of this line and the indication of which file the RSpec configuration belongs in? We don't indicate what file the configuration should go in for any of the other test libraries, and I have seen plenty of people put this configuration directly in the rails_helper or spec_helper. It is worth noting also that you are the sixth person (only looking at open PRs) to suggest an edit to this line (#987, #1025, #1081, #1085, and #1091). Perhaps it is confusing more people than it is helping.

Reply:
I still think it's necessary to mention that. I feel like a lot of new Ruby developers are pointed to Factory Bot in tutorials and might not be aware where that file would need to be required.  Thoughts?